charts/controller/templates/controller-ingress-rule-healthz-9090.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.global.experimental_native_ingress }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  namespace: "deis"
+  name: "controller-api-server-healthz"
+  labels:
+    app: "controller"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  rules:
+  - host: deis.{{ .Values.platform_domain }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: deis-controller
+          servicePort: 9090
+{{- end }}
```

Comment:
Not sure I understand this rule. afaik, the only thing that (optionally) uses 9090 is the router.

(TMI: Router is set up to guarantee PROXY protocol is never used on 9090, _regardless_ of whether it's used on 80 and 443. This is to allow load balancers to perform HTTP-based healthchecks. Healthchecks, in an ELB at least, don't support PROXY protocol.)

Back to the matter at hand... I could be wrong, but I am pretty sure controller's container doesn't listen on 9090 at all.